flocker/control/script.py

```diff
@@ -0,0 +1,40 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script for starting control service server.
+"""
+
+from twisted.python.usage import Options
+from twisted.internet.endpoints import TCP4ServerEndpoint
+
+from .httpapi import create_api_service
+from ..common.script import (
+    flocker_standard_options, FlockerScriptRunner, main_for_service)
+
+
+@flocker_standard_options
+class ControlOptions(Options):
+    """
+    Command line options for ``flocker-control`` cluster management process.
+    """
+    optParameters = [
+        ["port", "p", 4523, "The port to listen on.", int],
```

Comment:
Well... except we're going to have a very specific security policy we want, which is *not* expressable in the string format at the moment. I suggest considering that when we've actually implemented the TLS policy and see if that level of configuration makes sense.